docs/Caching.md

```diff
@@ -1,17 +1,16 @@
 # Backend Caching 
-cBioPortal provides the option of caching information on the backend as a means for improving performance. Without caching, each query including repeated queries, will query the database, process the returned data, and construct a response. This may lead to performance issues as the entire process is rather costly, especially for queries on larger studies. With caching turned on, the query response will be stored during the initial query. If the same query is made, the response will be taken directly from cache instead of having to be reconstructed. 
+cBioPortal provides the option of caching information on the backend to improve performance. Without caching, every time a request is received by the backend, a query is sent to the database system for information, and the returned data is processed to construct a response. This may lead to performance issues as the entire process can be rather costly, especially for queries on larger studies. With caching turned on, query responses can be taken directly from the cache if they have alredy been constructed. They only would be constructed for the initial query.
```

Comment:
Typo:

> With caching turned on, query responses can be taken directly from the cache if they have alredy been constructed.

With caching turned on, query responses can be taken directly from the cache if they have **already** been constructed.

Reply:
Rephrase:

> They only would be constructed for the initial query.

They would only be constructed for the initial query.

Reply:
changes made